modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,263 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
 In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
 
+[id="pipelines-new-features-1-8_{context}"]
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an {product-title} cluster that is running on ARM hardware. This includes support for `ClusterTask` resources and the `tkn` CLI tool.
```

Comment:
@rolfedh we need to add one known issue to ARM support:
https://issues.redhat.com/browse/SRVKP-2375 i.e. the `jib-maven` cluster task is known not to work on ARM based clusters.

Reply:
Added ARM and replaced this short known issue with the verbose one from the 1.7 release notes:

* When you run Maven and Jib-Maven cluster tasks, the default container image is supported only on Intel (x86) architecture. Therefore, tasks will fail on ARM, IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters. As a workaround, you can specify a custom image by setting the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
// issue # is unknown.
+
[TIP]
====
Before you install tasks that are based on the Tekton Catalog on ARM, IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) using `tkn hub`, verify if the task can be executed on these platforms. To check if `ppc64le` and `s390x` are listed in the "Platforms" section of the task information, you can run the following command: `tkn hub info task <name>`
// issue # is unknown.
====
// link:https://issues.redhat.com/browse/SRVKP-2375 for ARM issue